modules/machineset-yaml-gcp.adoc

```diff
@@ -80,10 +80,10 @@ endif::infra[]
             labels: null
             sizeGb: 128
             type: pd-ssd
+          gcpMetadata:
+            creationTimestamp: null
           kind: GCPMachineProviderSpec
           machineType: n1-standard-4
-          metadata:
-            creationTimestamp: null
```

Comment:
this should probably stay in the docs for a couple reasons. this is kinda wonky but, because that provider spec is actually a kube object under the hood, it will contain a `metadata` field and that `creationTimestamp: null` is what will be there. in the case of gcp, that `gcpMetadata` field is also there, but it serves a different purpose.

so, i think we should leave this because it will be there, and the other examples show this as well.